src/Stratis.Bitcoin.Features.MemoryPool/MempoolBehavior.cs

```diff
@@ -322,7 +322,7 @@ public override object Clone()
                 return;
             }
 
-            bool blocksOnly = !this.mempoolManager.mempoolSettings.RelayTxes;
+            bool blocksOnly = !this.connectionManager.ConnectionSettings.RelayTxes;
```

Comment:
why has RelayTxes moved to ConncectionSettings? I think it belongs to Mempool as it's where it's used.

Reply:
It is also used in the `Base` feature so if we define it in the (optional) Mempool feature it won's be available on the `Base` feature:

```
            var connectionParameters = this.connectionManager.Parameters;
            connectionParameters.IsRelay = this.connectionManager.ConnectionSettings.RelayTxes;
            connectionParameters.TemplateBehaviors.Add(new ChainHeadersBehavior(this.chain, this.chainState, this.initialBlockDownloadState, this.bestChainSelector, this.loggerFactory));
            connectionParameters.TemplateBehaviors.Add(new PeerBanningBehavior(this.loggerFactory, this.peerBanning, this.nodeSettings));
```